wrappers/azurekeyvault/azurekeyvault_acc_test.go

```diff
@@ -71,7 +71,7 @@ func TestAzureKeyVault_IgnoreEnv(t *testing.T) {
 	require.Equal(t, config["tenant_id"], s.tenantID)
 	require.Equal(t, config["client_id"], s.clientID)
 	require.Equal(t, config["client_secret"], s.clientSecret)
-	require.Equal(t, config["environment"], s.environment.Name)
+	require.Equal(t, config["environment"], azure.PublicCloud.Name)
```

Comment:
nit: If you provide "AzurePublicCloud" in place of `azure.PublicCloud.Name` in the test code we can drop the Azure/go-autorest dependency completely.